src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,24 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
     public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Logic logic = new Logic();
+        boolean isInRange = false;
+        boolean isDevidedByThree = false;
+        int var;
```

Comment:
Staraj się unikać nazwy zmiennej "var". W nowszych wersjach Javy słowa var można używać do deklarowania zmiennych, czyli np: `var x = 5;` i może to być nieco mylące